.github/CODEOWNERS

```
@@ -41,6 +41,7 @@ NOTICE                                           @awslabs/mcp-admins
 /src/aws-knowledge-mcp-server                    @awslabs/mcp-admins @awslabs/mcp-maintainers  @alexa-perlov @krokoko @scottschreckengaust # @tkaria @animebar
 /src/aws-location-mcp-server                     @awslabs/mcp-admins @awslabs/mcp-maintainers  @scottschreckengaust @theagenticguy # @scouturier
 /src/aws-msk-mcp-server                          @awslabs/mcp-admins @awslabs/mcp-maintainers  @elmoctarebnou @dingyiheng
+/src/aws-network-mcp-server      @juhala-aws @NetDevAutomate @taylaand
```

The admins and maintainers should be there too:
```suggestion
/src/aws-network-mcp-server                      @awslabs/mcp-admins @awslabs/mcp-maintainers  @juhala-aws @NetDevAutomate @taylaand
```